Source/Tools/UpgradeInstallation/build.sbt

```
@@ -1,12 +1,15 @@
 libraryDependencies ++= Seq(
-  "com.google.guava" % "guava"          % "31.0.1-jre",
-  "org.slf4j"        % "jcl-over-slf4j" % "1.7.33",
-  "org.slf4j"        % "slf4j-log4j12"  % "1.7.33",
-  "log4j"            % "log4j"          % "1.2.17",
+  "com.google.guava"         % "guava"            % "31.0.1-jre",
+  "org.slf4j"                % "jcl-over-slf4j"   % "1.7.33",
+  "org.apache.logging.log4j" % "log4j-slf4j-impl" % "2.17.1",
+  "org.apache.logging.log4j" % "log4j"            % "2.17.1",
   xstreamDep,
   "commons-configuration" % "commons-configuration" % "1.10",
   "commons-io"            % "commons-io"            % "2.11.0",
-  "commons-lang"          % "commons-lang"          % "2.6"
+  "commons-lang"          % "commons-lang"          % "2.6",
+  // Need these two jackson deps to allow processing log4j yaml config files.
+  "com.fasterxml.jackson.core"       % "jackson-databind"        % "2.11.3",
+  "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.11.3"
```

How about use `jacksonVersion`?

```suggestion
  "com.fasterxml.jackson.core"       % "jackson-databind"        % jacksonVersion,
  "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % jacksonVersion
```

---

The `jacksonVersion` created in `equellaserver/build.sbt` is not available here. 

`jackson-dataformat-yaml` is only used `UpgradeInstallation` so what about create a val here ?

---

Sounds like a plan. :+1: